cmd/jujud/agent/addons/addons_test.go

```diff
@@ -1,7 +1,7 @@
-// Copyright 2016 Canonical Ltd.
+// Copyright 2020 Canonical Ltd.
 // Licensed under the AGPLv3, see LICENCE file for details.
 
-package agent
+package addons_test
```

Comment:
Are we missing a package_test that actually registers the tests?

Reply:
ops, thanks for pointed out!